internal/convert/terratag_locals.go

```diff
@@ -34,11 +34,13 @@ func decodeTerratagLocals(locals Locals, s string) error {
 func encodeTerratagLocals(locals Locals) string {
 	ret := "{"
 
-	// Return it in ordered manner for order consistency (primarly when running tests).
-	var keys []string
+	// Return it in ordered manner for order consistency (primarily when running tests).
+	keys := []string{}
```

Comment:
Is this related to the versions upgrade?

Reply:
As part of the upgrade I had to replace the linter. So there are linter fixes as well.